### PR TITLE
[MXNET-67] Sync master with v1.1.0 branch

### PR DESCRIPTION
**Pull request by yzhliu**
I have went through the commits and checked all the changes on v1.1.0 branch has been merged into master, except the LICENSE and NEWS below.

But I did see some commits that were pushed into v1.1.0 directly (without PR against either v1.1.0 nor master), thus ping @eric-haibin-lin for double check.

The commits:
```
commit 31104c9d4b050883467f45f8bf9a164acb93976f
Author: Haibin Lin <linhaibin.eric@gmail.com>
Date:   Tue Feb 6 15:22:53 2018 -0800

    Update NEWS.md


commit 8b3c9ebb7bb4a9e8ee88e7222a718f7fa1c9a6be (tag: 1.1.0.rc0)
Author: Haibin Lin <linhaibin.eric@gmail.com>
Date:   Sat Jan 27 23:23:13 2018 -0800

    Update NEWS.md

commit 3ba84d83105bbc8825ac858f4c9cf81f9ca03d18
Author: Haibin Lin <linhaibin.eric@gmail.com>
Date:   Sat Jan 27 23:18:11 2018 -0800

    Update KEYS


commit 9a5819687f16ea7cd611bca7b4bcb809d4186d9d
Author: Haibin Lin <linhaibin.eric@gmail.com>
Date:   Sat Jan 27 18:18:35 2018 -0800

    update news.md (#191)
    
    * Update NEWS.md
    
    * Update README.md


commit 4c17c030208c67b2f68808d12d5a79996cfaf4ba
Author: Haibin Lin <linhaibin.eric@gmail.com>
Date:   Sat Jan 27 18:22:50 2018 -0800

    Bump 1.1 (#192)
    
    * bump
    
    * also update base.h
    
    * revert website changes
    
    * Update index.html
```

### CODE REVIEW

**Comment by eric-haibin-lin** [comment]
Looks good if README is also updated.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Apache MXNet (incubating) for Deep Learning
 =====
 
-[![Build Status](https://builds.apache.org/job/incubator-mxnet/job/master/badge/icon)](http://jenkins.mxnet-ci.amazon-ml.com/job/incubator-mxnet/job/master/)
+[![Build Status](https://builds.apache.org/job/incubator-mxnet/job/master/badge/icon)](https://builds.apache.org/job/incubator-mxnet/job/master/)
```
Comment (marcoabreu) [inline README.md]: Please revert this change. builds.apache.org is the old server
